media-sound/mumble/mumble-9999.ebuild

```diff
@@ -64,97 +65,42 @@ BDEPEND="
 	virtual/pkgconfig
 "
 
-# NB: qmake does not support multilib but it's fine to configure
-# for the native ABI here
 src_configure() {
-	myuse() {
-		[[ -n "${1}" ]] || die "myuse: No use option given"
-		use ${1} || echo no-${1}
-	}
 
-	local conf_add=(
-		bundled-celt
-		no-bundled-opus
-		no-bundled-speex
-		no-embed-qt-translations
-		no-server
-		no-update
-		$(myuse alsa)
-		$(myuse dbus)
-		$(usex debug 'symbols debug' release)
-		$(myuse g15)
-		$(usex jack '' no-jackaudio)
-		$(myuse opus)
-		$(myuse oss)
-		$(myuse portaudio)
-		$(myuse pulseaudio)
-		$(myuse rnnoise)
-		$(usex speech '' no-speechd)
-		$(usex zeroconf '' no-bonjour)
+	local mycmakeargs=(
+		"-Dbundled-celt=ON"
+		"-Dbundled-opus=OFF"
+		"-Dbundled-speex=OFF"
+		"-Doverlay=ON"
+		"-Dtranslations=OFF"
```

Comment:
Any reasons why you set this to `OFF`? Does creating the translations need any additional dependencies?

Reply:
It is simply mirroring the old settings in the ebuild.
The old flag was "no-embed-qt-translations".

I don't remember if I tried turning it on or not.

But I guess we can have a discussion if we should make this configurable or not if needed.

Reply:
Ah, okay. I've just realised that with that setting, mumble is english-only. Setting it to `ON` makes it translated (or English with `LC_ALL=C`). However, since no additional translation-files are installed, I assume they are integrated into the binary.
I will re-check with mumble-1.3.3 tomorrow to see how that behaves.
However, it might be that integrating the translation files might require `dev-qt/linguist-tools` as dependency.

Reply:
Okay, I've re-checked mumble-1.3.3: While it is true that the ebuild requests the translations not to be embedded, this doesn't seem to be honored. `lrelease` from `dev-qt/linguist-tools` is called during the build for each language and the resulting binary displays the texts translated. Since the emerge doesn't install any translation files, I assume that the texts are inside the binary (tried `strings` command but didn't find anything. Did not want to investigate how Qt's resource system (qrc) works in detail).
With the CMake-based build system, the configuration option is actually taken into account.
I'd prefer mumble to simply integrate the translations. I wouldn't know how to force them to separate files.

Reply:
According to the CMakeLists.txt file `-Dtranslation` triggers other tranlations than english. So please introduce a `nls` USE flag and change the option to `-Dtranslation="$(usex nls)"`